documentation/SDKs/javascript/README.md

```diff
@@ -7,352 +7,264 @@
 ### Installing the JavaScript SDK for Nether
 
 ## Usage
-
-### Analytics
-
-#### Active sessions - daily
-You can use the ***Nether.Analytics.GetActiveSessionsDaily*** method to get the total active sessions per day. Returns an array of ActiveSession
-##### Example usage
-```javascript
-var sessions = Nether.Analytics.GetActiveSessionsDaily();
-console.log(sessions[0].ActiveSessions + ' for ' + sessions[0].Day + '-' + sessions[0].Month + '-' + sessions[0].Year);
-```
-#### Active sessions - monthly
-You can use the ***Nether.Analytics.GetActiveSessionsMonthly*** method to get the total active sessions per month. Returns an array of ActiveSession
-##### Example usage
-```javascript
-var sessions = Nether.Analytics.GetActiveSessionsMonthly();
-console.log(sessions[0].ActiveSessions + ' for ' + sessions[0].Month + '-' + sessions[0].Year);
+To use the nether SDK include the nether script in your project. Reference the script in the webpage using 
+```html
+<script src="/scripts/nether.js"></script>
+```
+```javascript
+var config = {
+        netherClientId: '<client Id>',
+        netherClientSecret: '<client secret>',
+        facebookAppId: '<facebook app Id>',
+        netherBaseUrl: '<nether url>'
+    }
+    nether.init(config, facebookInitailised, netherInitialised);
```

Comment:
@stuartleeks @krist00fer  If we do want to secure the analytics endpoint then I think this should be changed to be an API within Nether which is secured in the same way as the other API's, that way not exposing the event hub end point in the JS which could potentially be used maliciously.